test/server.js

```diff
@@ -1,8 +1,10 @@
 // Quick start
+const path = require("path");
 
 // Using non-standard port
 const port = 2121;
-const homeDir = require("os").homedir() + "/code/nodejs/ftp-deploy/test/remote";
+// const homeDir = require("os").homedir() + "/code/nodejs/ftp-deploy/test/remote";
+const homeDir = path.join(__dirname, "/remote");
```

Comment:
the pattern above is generic and works on e.g. windows. why did you need to change it?